### PR TITLE
Add an OpenLDAP (slapd) role

### PR DESCRIPTION
**Pull request by OdyX**
* This PR is a : New feature

- [x] Documentation is written
- [ ] `parameters.yml.dist` is updated
- [x] `playbook.yml.dist` is updated
- [x] [Changelog](https://github.com/liip/drifter/blob/master/CHANGELOG.md) was updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ For migration information, you can always have a look at https://liip-drifter.re
 ### Changed
 
 - Django role: generate ALLOWED_HOSTS file to comply with Django >= 1.9.11
+- Add an OpenLDAP (slapd) role
```
Comment (sephii) [inline CHANGELOG.md]: Put this in the "Added" section

File: provisioning/roles/openldap/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+- name: Preseed slapd configuration options through debconf
+  debconf: name=slapd question={{ item.question }} value={{ item.value }} vtype={{ item.type }}
+  with_items:
+    - { question: slapd/internal/adminpw, value: admin, type: password }
```
Comment (sephii) [inline provisioning/roles/openldap/tasks/main.yml]: I think you should either make the password configurable, or add the credentials to the documentation. Also what's the difference between "adminpw" and "password1"?

File: docs/roles/others.md [hunk 3]
```diff
@@ -15,6 +15,15 @@ Install NodeJS and NPM.
 * **nodejs_version** : The version to install, currently supports 6.x, 5.x, 4.x, 0.12 and 0.10
 * **nodejs_distro** : Is automatically set to either 'jessie' or 'wheezy' based on available information, you can also put an Ubuntu codename here.
 
+# OpenLDAP
+
+Install an OpenLDAP (slapd) server
```
Comment (sephii) [inline docs/roles/others.md]: Add some useful info here: on what port can I reach it, using which credentials, etc.

File: provisioning/roles/openldap/tasks/main.yml (hunk 2 above)
Comment (OdyX) [inline provisioning/roles/openldap/tasks/main.yml]: That was your code; I can change it. :-)

Comment (OdyX) [inline provisioning/roles/openldap/tasks/main.yml]: https://sources.debian.net/src/openldap/2.4.44%2Bdfsg-3/debian/slapd.templates/#L87

&

https://sources.debian.net/src/openldap/2.4.44%2Bdfsg-3/debian/slapd.templates/#L66

**Comment by sephii** [comment]
Thanks! :grapes:

**Pull request merged**